package.json

```diff
@@ -6,7 +6,6 @@
     "name": "Clockwork Labs",
     "email": "no-reply@clockworklabs.io"
   },
-  "type": "module",
```

Comment:
I'm a bit confused with all the other module types and NPM options, so sorry if this is a stupid question, but will this still allow to use it as a module from the terminal?

Reply:
![image](https://user-images.githubusercontent.com/46858886/229156239-6aed12ad-aa90-415a-a911-970e4abe95de.png)
It should not change that functionality